content/en/docs/zero-code/obi/setup/kubernetes-helm.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Deploy OBI in Kubernetes with Helm
+linkTitle: Helm chart
+description: Learn how to deploy OBI as a Helm chart in Kubernetes.
+weight: 3
+---
+
+{{% alert title="Note" %}}
+
+For more details about the diverse Helm configuration options, check out the
+[OBI Helm chart options](https://github.com/grafana/beyla/blob/main/charts/beyla/README.md)
```

Comment:
I assume this will also point to `https://github.com/open-telemetry/opentelemetry-helm-charts/`

Reply:
will fix that but that file will be hidden to begin with